packages/plc/build.js

```diff
@@ -13,10 +13,11 @@ require('esbuild')
     platform: 'node',
     assetNames: 'src/static',
     external: [
-      './node_modules/better-sqlite3/*',
-      '../../node_modules/better-sqlite3/*',
+      'better-sqlite3',
```

Comment:
I couldn't find any downside to this change (e.g. dev-env still seems happy), and it played nicer for me when consuming the build from outside the monorepo.

Reply:
ah nice. that's probably fine. my whole method with these was guess & check anyway 😅 so id say if it builds & runs you're good